src/main/java/com/revature/ams/Member/MemberService.java

```diff
@@ -1,21 +1,56 @@
 package com.revature.ams.Member;
 
 import com.revature.ams.util.interfaces.Serviceable;
-import com.revature.ams.Member.Member;
 
+import java.util.ArrayList;
+import java.util.List;
+
+// TODO: REVIEW ME
 public class MemberService implements Serviceable<Member> {
+    private List<Member> memberList = new ArrayList<>();
```

Comment:
Regarding the use of lists and arrays, in the context of the project what scenario presented itself that made have to switch between the two?

Reply:
Nothing aside from as a data-structure lists are more adaptable to size changes. I will be adjusting flights to utilize Lists shortly as well.